forms/common-themes/src/main/resources/theme/base/admin/resources/partials/federated-generic.html

```diff
@@ -37,6 +37,9 @@
                     <input class="form-control" type="text" data-ng-model="instance.config[ option ]" >
                 </div>
             </div>
+
+            <kc-provider-config realm="realm" config="instance.config" properties="providerFactory.properties"></kc-provider-config>
```

Comment:
I deliberately added this kc-provder-config after the "conventional" option rendering to remain backwards compatible.